src/index.js

```diff
@@ -1,7 +1,47 @@
+const client = require('prom-client');
 const config = require('config');
 const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
+const Storage = require('./statuscakedata');
 
-this.interval = setInterval(fetchStatuCakeData, config.statuscake.interval);
+const regex = /\s?([-])\s?/;
+
+const getTagsFromTitle = Title => Title.split(regex);
+
+const loadtimeGauge = new client.Gauge({
+  name: 'frontend_pagespeed_loadtime_seconds',
+  help: 'Loadtime of your page',
+  labelNames: ['customer', 'service', 'device', 'version'],
+});
+const filesizeGauge = new client.Gauge({
+  name: 'frontend_pagespeed_filesize_bytes',
+  help: 'Filesize of your page',
+  labelNames: ['customer', 'service', 'device', 'version'],
+});
+const requestGauge = new client.Gauge({
+  name: 'frontend_pagespeed_request_count',
+  help: 'Number of request',
+  labelNames: ['customer', 'service', 'device', 'version'],
+});
+
+setInterval(() => {
+  const date = (100, Date.now());
```

Comment:
What's this notation? 👀

Reply:
Thats was for the example. Erased now. Thanks.